README.md

```diff
@@ -151,9 +151,26 @@ on the command line:
 ```bash
 mkdir build && cd build
 qmake -r ../librepcb.pro
-make -j 8
+make -j8
 ```
 
+To speed up subsequent compilation steps, it's recommended to install and
+enable [ccache](https://ccache.dev/) (this will work on Qt 5.9.2 and newer):
+
+```bash
+qmake -r ../librepcb.pro CONFIG+=ccache
+```
+
+By default, all binaries will be linked statically using vendored libraries. If
+you would prefer to unbundle some libraries, set the `UNBUNDLE` variable:
+
+```bash
+qmake -r ../librepcb.pro UNBUNDLE+=quazip
```

Comment:
Would it be possible to add an option to unbundle everything?

Otherwise LGTM, I did not test however.

Reply:
I added support for `UNBUNDLE=all`.